src/ahbicht/expressions/expression_resolver.py

```diff
@@ -101,11 +102,23 @@ def package(self, tokens: List[Token]) -> Awaitable[Tree]:
         """
         try to resolve the package using the injected PackageResolver
         """
-        return self._package_async(tokens)
-
-    async def _package_async(self, tokens: List[Token]) -> Tree[Token]:
-        resolved_package = await self._resolver.get_condition_expression(tokens[0].value)
+        package_key_token = [t for t in tokens if t.type == "PACKAGE_KEY"][0]
+        repeatability_tokens = [t for t in tokens if t.type == "REPEATABILITY"]
```

Comment:
Kannst du nochmal sagen warum du bei dem Package Key noch den 0ten Index aufrufen muss, aber nicht bei der repeatability? In deinem Test sehen sie analog aufgebaut aus: 
![grafik](https://user-images.githubusercontent.com/73470827/155686018-7178b557-0a23-4aab-89ea-607734577009.png)
Bzw. gibt es schon einen Test vom Parsing bis zur Repeatability? Also, dass aus einem Token wie hier die Repeatability gelesen wird?

Reply:
Hier erstmal ein ausführlicherer kommentar zum index 0: b4f35d2d27338890815b779128dfc39e329a6066

Reply:
Hier ist ein Test, der zeigt, dass das genauso ankommt:
https://github.com/Hochfrequenz/ahbicht/blob/c45beba29796aaa905f5c929c97d9fdde51c8dfe/unittests/test_condition_parser.py#L458